canvasapi/discussion_topic.py

```diff
@@ -30,6 +33,8 @@ def _parent_type(self):
         """
         if hasattr(self, "course_id"):
             return "course"
+        elif hasattr(self, "context_code"):
+            return "course"
```

Comment:
As noted above, I think it's possible for the `context_code` to be of type other than `"course"`.